assets/Css/ComponentsStyles.css

```diff
@@ -212,3 +212,55 @@ main {
   width: 4rem;
 }
 
+/* Alert Basic */
+.alert__wrapper--basic{
+  display: flex;
+  align-items: center;
+  width: fit-content;
+  padding:0.5rem;
+  border-radius: 0.5rem;
+  margin: 1rem;
+}
+.alert__img{
+  width: 1.5rem;
+  height: 1.5rem;
+}
+.alert__text{
+  font-size:0.8rem;
+  margin-left:0.5rem;
+}
+/* Alert Modified */
+.alert__wrapper--modified{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-width: 10rem ;
+  max-width: 50rem;
+  padding:0.5rem;
+  border-radius: 0.5rem;
+  margin: 1rem;
+  text-align: justify;
+}
+.alert__title{
+  font-size:1rem;
+  font-weight: 900;
+  padding:0.5rem;
+}
+
+/* Alert Color */
+.alert--success{
+  background-color: rgb(0, 199, 0);
```

Comment:
use variables instead of hard coded color values

Reply:
okay, I will convert it into a variable.
ThankYou